WalletWasabi/Tor/TorMonitor.cs

```diff
@@ -46,12 +43,8 @@ protected override async Task ActionAsync(CancellationToken token)
 					{
 						if (torEx.RepField == RepField.HostUnreachable)
 						{
-							Uri baseUri = new Uri($"{FallBackTestRequestUri.Scheme}://{FallBackTestRequestUri.DnsSafeHost}");
```

Comment:
I'm not sure why this was added. Some context would be helpful whether I'm not changing behavior here somehow.

Reply:
I don't know. @lontivero ?

Reply:
We don't know why, can you ut back the original behavior with a comment, that we don't know why is it here.

Reply:
Why not rather test it?

Reply:
How? I don't know what to test.

Reply:
So I would simply try to test that TorMonitor works as expected. To me it looks like we just need to test common scenario where we kill Tor and then simulate `RepField.HostUnreachable` case. Am I correct?

Reply:
Isn't this related when with http or https in the config file at values: 
```
  "MainNetBackendUriV3": "http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion/",
  "TestNetBackendUriV3": "http://testwnp3fugjln6vh5vpj7mvq3lkqqwjj3c2aafyu7laxz42kgwh2rad.onion/",
  "MainNetFallbackBackendUri": "https://wasabiwallet.io/",
  "TestNetFallbackBackendUri": "https://wasabiwallet.co/",
```

Reply:
The code trying to fix :// for some reason.

Reply:
Okay I made a few tests and I think it is a leftover. If it is wrong the bug will turn out anyway.

Reply:
Looks like a format fix. If the code works with our backend, then whatever, remove it.